gradio/utils.py

```diff
@@ -907,7 +904,7 @@ def tex2svg(formula, *args):
     DPI = 300
     plt.rc("mathtext", fontset="cm")
     fig = plt.figure(figsize=(0.01, 0.01))
-    fig.text(0, 0, r"${}$".format(formula), fontsize=FONTSIZE)
+    fig.text(0, 0, f"${formula}$", fontsize=FONTSIZE)
```

Comment:
```suggestion
    fig.text(0, 0, fr"${formula}$", fontsize=FONTSIZE)
```

Reply:
It doesn't matter, since $ isn't an escape sequence...

Reply:
I think you're right lol